pkg/scheduler/core/assignment_test.go

```diff
@@ -580,8 +580,8 @@ func Test_dynamicScale(t *testing.T) {
 			},
 			want: []workv1alpha2.TargetCluster{
 				{Name: ClusterMember1, Replicas: 7},
-				{Name: ClusterMember2, Replicas: 8},
```

Comment:
This case is not right, as cluster 2 already disappeared, it should not be in the schedule result.

Reply:
If this is a legacy issue, please track it by another issue.

Reply:
The calculation logic is as follows:
1. Calculate the number of instances that need to be reallocated: 24-4 = 20.
2. Sort the member clusters by weight: m4:m3:m1=12:8:4
3. Allocate the number of instances: m4：20/24*12=10, m3:20/24*8=6, m1:4+20/24*4=7
4. Distribute the remainder 1, resulting in m4：11, m3:6, m1:7